### PR TITLE
 implement SellerTradeParty->Description (BT-33) other legal information

### PR DESCRIPTION
**Pull request by PeterSawatzki-MGGM**

### CODE REVIEW

File: ZUGFeRD/InvoiceDescriptor.cs [hunk 1]
```diff
@@ -164,6 +164,7 @@ public class InvoiceDescriptor
         public Profile Profile { get; internal set; } = Profile.Basic;
 
         /// <summary>
```
Comment (stephan-nowecor) [inline ZUGFeRD/InvoiceDescriptor.cs:166]: Can you please add documentation here?

File: ZUGFeRD/InvoiceDescriptor.cs [hunk 2]
```diff
@@ -164,6 +164,7 @@ public class InvoiceDescriptor
...
+        public string Name { get; set; }
```
Comment (stephanstapel) [inline ZUGFeRD/InvoiceDescriptor.cs:167]: Can you please add documentation here?

File: ZUGFeRD/InvoiceDescriptor22CIIWriter.cs [hunk 3]
```diff
@@ -1266,7 +1266,7 @@ private void _writeOptionalParty(ProfileAwareXmlTextWriter writer, PartyTypes pa
                 case PartyTypes.BuyerTradeParty:
                     break;
                 case PartyTypes.ShipToTradeParty:
-                    if ((this.Descriptor.Profile != Profile.Extended) && (this.Descriptor.Profile != Profile.XRechnung1) && (this.Descriptor.Profile != Profile.XRechnung)) { return; } // extended, XRechnung1, XRechnung profile only
+                    if ((this.Descriptor.Profile != Profile.Extended) && (this.Descriptor.Profile != Profile.Comfort) && (this.Descriptor.Profile != Profile.XRechnung1) && (this.Descriptor.Profile != Profile.XRechnung)) { return; } // extended, XRechnung1, XRechnung profile only
```
Comment (stephanstapel) [inline ZUGFeRD/InvoiceDescriptor22CIIWriter.cs:1269]: can you please remove 'Comfort' here? This is not compliant with the documentation

Comment (PeterSawatzki-MGGM) [reply ZUGFeRD/InvoiceDescriptor22CIIWriter.cs]: done

**Pull request merged**